### PR TITLE
imageless airgap

### PR DESCRIPTION
**Pull request by areed**

### CODE REVIEW

File: docs/distributing-an-application/airgapped-installations.md [hunk 1]
```diff
@@ -59,3 +59,10 @@ page.
 ## Adding Additional Nodes
 
 In order to add additional nodes to your cluster, just navigate to the Cluster page of the Admin Console, click the "Add Node" button, and follow the instructions there. For more detailed instructions see the [add nodes](/distributing-an-application/add-nodes/) page of the docs.
+
+## Imageless Airgap Installations
+Licenses with Airgap installations enabled can additionally be configured to offer imageless bundles to customers.
+When imageless downloads are enabled in the vendor portal, customers will see two links at the download URL you
```
Comment (divolgin) [inline docs/distributing-an-application/airgapped-installations.md]: I think we don't do second person pronouns.  It's in the contributing guide somewhere (linked from the README file)

**Review by divolgin — APPROVED** [review]
Maybe don't merge this until 2.11 is ready to be released?

**Comment by areed** [comment]
Removed "you" and will wait to merge.

**Comment by areed** [comment]
This is a secret feature

**Pull request closed without merging**